### PR TITLE
Add gemspec

### PR DESCRIPTION
**Pull request by camertron**
Eventually we'd like to distribute txgh as a gem. This is a step in the right direction.

@lumoslabs/platform

### CODE REVIEW

File: txgh.gemspec [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  s.add_dependency 'faraday_middleware', '~> 0.10'
+  s.add_dependency 'json', '~> 1.8'
+  s.add_dependency 'octokit', '~> 4.2'
+  s.add_dependency 'puma', '~> 2.15'
+  s.add_dependency 'parseconfig', '~> 1.0'
+  s.add_dependency 'sinatra', '~> 1.4'
+  s.add_dependency 'sinatra-contrib', '~> 1.4'
+
+  s.require_path = 'lib'
+  s.files = Dir['{lib,spec}/**/*', 'Gemfile', 'README.md', 'Rakefile', 'txgh.gemspec']
```
Comment (jdoconnor) [inline txgh.gemspec]: do we want to include gemfile and rakefile in the gem?

Comment (camertron) [inline txgh.gemspec]: Is it common practice to include the Gemfile?

**Comment by camertron** [comment]
Ok with you @jdoconnor?

**Comment by jdoconnor** [comment]
+1

**Pull request merged**